src/ui/pages/error-page.svelte

```diff
@@ -115,7 +121,11 @@
   {/snippet}
 
   {#snippet message()}
-    {getTranslatedErrorMessage()}
+    {#if isSandbox && error.underlyingErrorMessage}
+      {error.underlyingErrorMessage}
```

Comment:
We have some e2e broken because of this change. I can move this to another PR since it affects non-taxes errors too.

I also think we should have storybooks stories covering this

Reply:
I dropped this change, we can include this as part of a different PR